app/assets/javascripts/components/bulkUploadImport.jsx

```diff
@@ -333,131 +380,310 @@ class BulkUploadImport extends React.Component {
+                  <div className='row input-row'>
+                    <div className='col project-list no-padding s8 tooltipped' data-position="top" data-delay="50" data-tooltip="Name of experiment or project">
+                      <select ref="projectSelect" disabled={(this.state.disableProjectSelect ? 'disabled' : '')} className="projectSelect" id="sample" onChange={ this.handleProjectChange } value={this.state.project}>
+                        <option disabled defaultValue>{this.state.project}</option>
+                        { this.state.allProjects.length ?
+                          this.state.allProjects.map((project, i) => {
+                            return <option ref= "project" key={i} id={project.id} >{project.name}</option>
+                          }) : <option>No projects to display</option>
+                        }
+                      </select>
+                      {
+                        (this.state.errors.project) ?
+                          <div className='field-error'>
+                            {this.state.errors.project}
+                          </div> : null
+                      }
+                    </div>
+                    <div className='col no-padding s4'>
+                      <button type='button' onClick={this.toggleNewProjectInput}
+                              className='new-project-button new-button skyblue-button tooltipped'
+                              data-position="right" data-delay="50" data-tooltip="Add your desired experiment or project name">
+                        <i className='fa fa-plus'/>
+                        <span>
+                          New project
+                        </span>
+                      </button>
+                    </div>
+                    <div className='col no-padding s12 new-project-input hidden'>
+                      <input type='text' onBlur={ (e) => {
+                        if (e.target.value.trim().length) {
+                          this.handleProjectSubmit();
                         }
-                  </select>
-                  <label>Default host genome</label>
+                        $('.new-project-button').click();
+                      }} ref='new_project' onFocus={ this.clearError } className='browser-default' placeholder='Input new project name' />
+                      {
+                        (this.state.errors.new_project) ?
+                          <div className='field-error'>
+                            {this.state.errors.new_project}
+                          </div> : null
+                      }
+                    </div>
+                  </div>
+                </div>
+                <div className='field'>
+                  <div className='row'>
+                    <div className='col field-title no-padding s5 tooltipped'
+                         data-position="top" data-delay="50"
+                         data-tooltip='This would be subtracted by the pipeline'>
```

Comment:
This tooltip is not showing up for me. 
And text should be "This will be subtracted by the pipeline" ?